examples/performance-debugging/performance-debugging.cpp

```diff
@@ -343,8 +343,9 @@ int main(int argc, char *argv[])
     // Parametrize the benchmark here
     // Pick a value type
     using vtype = double;
+    using itype = int;
```

Comment:
Maybe for uniformization, also use ValueType and IndexType here.

Reply:
Down here, line `354` the reduction_factor also needs adaptation, although it's actually quite low.

Reply:
Up here, line `327`, missing Dense change for the `print_vector` function.